package.json

```diff
@@ -11,7 +11,8 @@
     "rollback": "scripts/rollback",
     "latest": "scripts/latest",
     "seed": "scripts/seed",
-    "lint": "scripts/lint"
+    "lint": "scripts/lint",
+    "knex": "knex"
```

Comment:
Yeah, we don't need this here @meghanprestemon, if you haven't already please add a commit removing this line, thanks!